test/test_unit.jl

```diff
@@ -1737,6 +1737,124 @@ end
     @test isapprox(mu_control(u, equations_parabolic, T_ref, R_specific, C, mu_ref),
                    1.803e-5, atol = 5e-8)
 end
+
+# Velocity functions are present in many equations and are tested here
+@testset "Velocity functions for different equations" begin
+    gamma = 1.4f0
+    rho = pi * pi
+    pres = sqrt(pi)
+    v1, v2, v3 = pi, exp(1.0f0), exp(pi) # use pi, exp to test with non-trivial numbers
```

Comment:
```suggestion
    gamma = 1.4
    rho = pi * pi
    pres = sqrt(pi)
    v1, v2, v3 = pi, exp(1.0), exp(pi) # use pi, exp to test with non-trivial numbers
```

It's a bit cleaner to use `Float64` everywhere in these tests instead of mixing `Float32` and `Float64`.